tests/test_libappimage.cpp

```diff
@@ -569,7 +572,7 @@ bool test_compare_bytes(const char* buf1, const char* buf2, int size) {
 
 TEST_F(LibAppImageTest, appimage_type2_digest_md5) {
     char digest[16];
-    char expectedDigest[] = {-20, 92, -89, 99, -47, -62, 14, 36, -5, -127, 65, -126, 116, -41, -33, -121};
+    char expectedDigest[] = {-75,  -71,  106,  -93,  122,  114,  7,  127,  -40,  10,  -115,  -82,  -73,  115,  -19,  1};
```

Comment:
Why did the test AppImage size increase? I thought you replaced some file by a symlink. Any explanation for that? I'm curious, to be honest. ~88kiB is quite much.

Reply:
Good question. I just unpacked, create the required libs, moved the desktop file and created the symlink. I'll take another look at it.

Reply:
The decompressed AppImages folder sizes differ in only 6kb. But it seems that the new version of appimagetool creates a bit larger files than the one that I used when created the first file.

Reply:
We should investigate that, but it won't prevent us from merging this PR.

Reply:
Fixed.